apps/bfd-data-fda/src/test/java/gov/cms/bfd/data/fda/utility/AppTest.java

```diff
@@ -0,0 +1,57 @@
+package gov.cms.bfd.data.fda.utility;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.any;
+
+import org.junit.jupiter.api.Test;
+import org.mockito.MockedStatic;
+import org.mockito.Mockito;
+import org.mockito.stubbing.Answer;
+
+public final class AppTest {
```

Comment:
Missing javadocs in this class. Not sure why checkstyle didn't catch this... your build should have failed. I'll need to look into adjusting the config maybe.

Reply:
done